artifacts/examples/image-security-policy-crd.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
shall we move the definition one level up of example?

Reply:
done